packages/flutter_tools/lib/src/protocol_discovery.dart

```diff
@@ -60,10 +59,14 @@ class ProtocolDiscovery {
 
   void _handleLine(String line) {
     Uri uri;
-    final int index = line.indexOf(_prefix + 'http://');
-    if (index >= 0) {
+
+    final RegExp r =
+        new RegExp('${RegExp.escape(serviceName)} (.*) (http://[^ \n]+)');
```

Comment:
Nit: no need to wrap - we don't wrap at 80 chars

Reply:
Yep, I was switching to VSCode instead of vim, fault of mine